README.md

```diff
@@ -51,15 +49,13 @@ You can leave the Beta testing program at any time:
 
 App data is wiped from the device when a user logs out of the app. If the user is logged in when the account is deactivated, then within one minute the system logs the user out, and as a result all app data is wiped from the device.
 
-### Can I connect to multiple Mattermost servers using the mobile apps?
-
-At the moment, we only support connecting to one server at a time. If you need to connect to multiple servers, please [upvote the feature request](https://mattermost.uservoice.com/forums/306457/suggestions/10975938) so we can track demand for it. 
+### Will there be second generation apps available for tablets?
```

Comment:
Propose remove this question. tablet support was built in v2

Reply:
Agree with this comment

Reply:
removed